expediaRequester/requester.py

```diff
@@ -6,35 +6,37 @@ class ExpediaRequester(object):
     base_url = "http://terminal2.expedia.com:80/x/"
 
     def __init__(self, key):
-        self.apiKey = key
+        self.apikey = key
 
-    def get(self, URL):
-        response = requests.get(URL)
+    def get(self, URL, parameter = None):
+        response = requests.get(URL, params = parameter)
```

Comment:
Can you integrate the None handling in this method instead of each and every method. This would make the implementation of other methods pretty simple.
Just remove the elements with None value from the dictionary